csrc/scheduler/reduction_utils.cpp

```diff
@@ -30,9 +31,11 @@ TensorView* scheduleReductionTV(
   // Inner here though is only relative to the other axis. When
   // rparams.fastest_dim == false, the reduction axis is logically outside the
   // iteration axis.
-  const int iter_axis = 0;
+  const int has_outmost_dim_sharded = isSharded(reduction_tv);
```

Comment:
Please add a comment about this. For example, we should note that we assume only the outermost domain can be parallelized with DIDx at this point and in that case this reduction scheduler only schedules the remaining domains while leaving the DIDx domain unchanged.